CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+* Version 1.5.2
+  Support custom URL comparison when adding new pages during crawling
+  Don't rescue Timeout error, so that Delayed Job can properly terminate hanging jobs
+  Fail early if Capybara doesn't initialize properly
+
 * Version 1.5.1
   Fixed deprecation warning (Thanks scott)
   Updated Poltergeist dependency
```

Comment:
look at the output using the "View" button, it's a big mess :see_no_evil: 

IMHO it should
- use `#` to set the versions apart
- use `*` if a given version has multiple features
- remove the `Version `, it's quite obvious and serves only to make it harder to scan-read.